File: src/kyu_6/triple_trouble_1/python/triple_trouble_1.py
def triple_double(num1, num2):
    n1str = str(num1)
    n2str = str(num2)
    for i in range(10):
        n = str(i)
        if n1str.count(n  *  3) > 0 and n2str.count(n  *  2) > 0:
            return 1
    return 0

File: src/kyu_6/triple_trouble_1/csharp/triple_trouble_1.cs
using System;
using System.Collections.Generic;

public class Kata {
    public static int TripleDouble(long num1, long num2) {
       string n1str = num1.ToString();
        string n2str = num2.ToString();

        for (int i = 0; i < 10; i++)
        {
            string n = i.ToString();
            if (n1str.Contains(n + n + n) && n2str.Contains(n + n))
            {
                return 1;
            }
        }

        return 0;
    }
}